openc3/lib/openc3/api/limits_api.rb

```diff
@@ -364,5 +364,34 @@ def _limits_group(group_name, action:, scope:, token:)
         TargetModel.set_packet(last_target_name, last_packet_name, packet, scope: scope)
       end
     end
+
+    # Gets an item. The code below is mostly duplicated from tlm_process_args in tlm_api.rb.
+    #
+    # @param target_name [String] target name
+    # @param packet_name [String] packet name
+    # @param item_name [String] item name
+    # @param scope [String] scope
+    # @return Hash The requested item based on the packet name
+    def _get_item(target_name, packet_name, item_name, scope:)
+      requested_item = nil
+      if packet_name == 'LATEST'
+        latest = -1
+        TargetModel.packets(target_name, scope: scope).each do |packet|
```

Comment:
This whole loop should be a CvtModel or TargetModel method to get the latest item.